### PR TITLE
LPD-15259 | Test for LPD-15117

### PR DESCRIPTION
**Pull request by jaimelr10**
**Description**: Test that we can select object definitions when there are more than 20 on schema creation

**Jira Ticket**: https://liferay.atlassian.net/browse/LPD-15259

### CODE REVIEW

**Comment by jaimelr10** [comment]
ci:test:playwright

File: modules/test/playwright/tests/headless-builder-web/schemas.spec.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	loginTest,
+	headlessBuilderPagesTest
+);
+
+test('can see all available object defitions on schema creation', async ({
+	apiHelpers,
+	applicationPage,
+	headlessBuilderPage,
+}) => {
+	const objectDefinitionArray = [];
```
Comment (4lejandrito) [inline modules/test/playwright/tests/headless-builder-web/schemas.spec.ts:23]: Call it `objectDefinitions`.

File: modules/test/playwright/tests/headless-builder-web/schemas.spec.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+					await apiHelpers.objectAdmin.deleteObjectDefinition(
+						objectDefinition.id
+					)
+				).status()
+			)
+			.toBe(204);
+	}
+
+	await headlessBuilderPage.goto();
+	await headlessBuilderPage.deleteApplication('My-app');
```
Comment (4lejandrito) [inline modules/test/playwright/tests/headless-builder-web/schemas.spec.ts:98]: I would do this via headless.

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

**Comment by jaimelr10** [comment]
ci:test:playwright

**Review by albertojml — APPROVED** [review]
LGTM!

**Comment by jaimelr10** [comment]
ci:test:playwright

**Comment by jaimelr10** [comment]
ci:forward

**Pull request closed without merging**